xml/System.Windows.Forms.DataVisualization.Charting/SeriesCollection.xml

```diff
@@ -56,7 +56,7 @@
   
  The `name` argument refers to the `Name` property of the object being added.  
   
- **Note** If you use this method to add <xref:System.Windows.Forms.DataVisualization.Charting.Series> objects to the <xref:System.Windows.Forms.DataVisualization.Charting.SeriesCollection> class, the associated <xref:System.Windows.Forms.DataVisualization.Charting.DataPoint> objects of the newly added <xref:System.Windows.Forms.DataVisualization.Charting.Series> object cannot have more than one Y-value per data point. To add a series that can handle multiple Y-values per data point, use the <xref:System.Windows.Forms.DataVisualization.Charting.SeriesCollection.Add%2A> System.Int32)?qualifyHint=False&autoUpgrade=True definition of this overloaded method.  
+ **Note** If you use this method to add <xref:System.Windows.Forms.DataVisualization.Charting.Series> objects to the <xref:System.Windows.Forms.DataVisualization.Charting.SeriesCollection> class, the associated <xref:System.Windows.Forms.DataVisualization.Charting.DataPoint> objects of the newly added <xref:System.Windows.Forms.DataVisualization.Charting.Series> object cannot have more than one Y-value per data point. To add a series that can handle multiple Y-values per data point, use the <xref:System.Windows.Forms.DataVisualization.Charting.SeriesCollection.Add(System.Int32)> definition of this overloaded method.  
```

Comment:
The link with Add(Int32) is broken; the method does not exist (see the earlier comment).

Reply:
I think this one should be removed also.